tox.ini

```diff
@@ -93,7 +95,8 @@ commands =
     mypy -p libqtile
     # also run the tests that require mypy
     python3 setup.py -q install
-    python3 -m pytest -W error -- test/test_check.py test/test_migrate.py
+# py310 currently fails with -W error, see: https://gitlab.gnome.org/GNOME/pygobject/-/issues/476
+    python3 -m pytest -- test/test_check.py test/test_migrate.py
```

Comment:
Do we get the same `ImportWarning` here, even if we're only running `test_check` and `test_migrate`?

Reply:
Yeah it happens at the very beginning during the imports. I did test it without this change so can confirm.